src/module/actor/sheet/base.ts

```diff
@@ -1061,6 +1061,12 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
                     {},
                     async (heightenedLevel, itemType, spell) => {
                         const createdItem = await createConsumableFromSpell(itemType, spell, heightenedLevel);
```

Comment:
Do it from this function:

1. Convert the parameters to a destructured object.
2. Add a `mystify` boolean property.